.github/workflows/ci.yml

```
@@ -14,85 +14,89 @@ jobs:
-        xcodebuild test \
-          -project Virgo.xcodeproj \
-          -scheme Virgo \
-          -destination '${{ matrix.destination }}' \
-          -configuration Debug \
-          -only-testing:VirgoUITests \
-          ONLY_ACTIVE_ARCH=NO \
-          CODE_SIGNING_REQUIRED=NO \
-          CODE_SIGNING_ALLOWED=NO
+      - name: Checkout code
+        uses: actions/checkout@v4
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)